app/components/Layout/Header.jsx

```diff
@@ -252,7 +252,7 @@ class Header extends React.Component {
 
     render() {
         let {active} = this.state;
-        let {currentAccount, starredAccounts, passwordLogin, passwordAccount, height} = this.props;
```

Comment:
This will cause the following error
![image](https://user-images.githubusercontent.com/2197479/36138021-bc97a214-10d2-11e8-9339-9fd2403cc74f.png)

Reply:
Thanks. Must have happened when I merged with staging. Had another strange change that didn't come over with it.